x-pack/plugins/rollup/public/plugin.ts

```diff
@@ -108,16 +99,9 @@ export class RollupPlugin implements Plugin {
     }
   }
 
-  start(core: CoreStart, plugins: RollupPluginStartDependencies) {
+  start(core: CoreStart) {
     setHttp(core.http);
     setNotifications(core.notifications);
     setEsBaseAndXPackBase(core.docLinks.ELASTIC_WEBSITE_URL, core.docLinks.DOC_LINK_VERSION);
-
-    const isRollupIndexPatternsEnabled = core.uiSettings.get(CONFIG_ROLLUPS);
-
-    if (isRollupIndexPatternsEnabled) {
-      initAggTypeFilter(search.aggs.aggTypeFilters);
-      initAggTypeFieldFilter(plugins.data.search.__LEGACY.aggTypeFieldFilters);
-    }
```

Comment:
Now that the agg type filters registry is gone, rollup plugin no longer has a dependency on `data`... I've updated the `kibana.json` accordingly